sources/procfs.go

```diff
@@ -675,6 +676,34 @@ func parseFileElements(path string, directoryDepth int) (name string, nodeName s
 	return name, nodeName, nil
 }
 
+func convertToBytes(s string) string {
+	if len(s) < 1 {
+		return s
+	}
+	new_s := ""
+	multiplier := float64(1)
+	switch finalChar := strings.ToUpper(s[len(s)-1:]); finalChar {
```

Comment:
@joehandzik just a question. Are they originally upper or lower case? Why do you have to force to the upper case?